src/components/Auth/AuthProvider.js

```diff
@@ -0,0 +1,146 @@
+import React, { useContext } from 'react';
+import { useLazyQuery } from '@apollo/client';
+import { useGoogleLogin } from 'react-use-googlelogin';
+import env from '../../utils/env';
+import { signInRed, signOutRed } from './state/loginReducer';
+import { storeInLocalStorage, deleteFromLocalStorage } from '../../utils/localStorage';
+
+import GET_USER_DETAILS from '../../bento/authProviderData';
+
+const AUTH_API = env.REACT_APP_AUTH_SERVICE_API;
+const GOOGLE_CLIENT_ID = env.REACT_APP_GOOGLE_CLIENT_ID;
```

Comment:
Can you add support for situation when "REACT_APP_GOOGLE_CLIENT_ID"  is undefined? 

For More info: https://github.com/CBIIT/bento-frontend/pull/672/commits/d94c384d4bde7d91078b15f5a8f3d12614856b89

Reply:
Made the change for this.